### PR TITLE
Community Demo Announcement Blog

### PR DESCRIPTION
**Pull request by cartersocha**
Put together an announcement blog for the community demo effort. Hopefully this will drive awareness of & engagement with the project. 

Tentative release date would be June 20th. Waiting on several small clean up PRs & docker improvements to reduce build time. 

I wasn't sure where to store the images so I just created a dedicated folder.

### CODE REVIEW

**Comment by cartersocha** [comment]
@mtwo - where should I be storing the photos to not break the checks? also could you take a quick look to review the material. 

Ideally we post this on Monday

File: content/en/blog/2022/demo-announcement.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+* Designed to work on docker locally
+* Uses redis cache
+* Auto-instrumentation using instrumentation libraries Tracing support for the gRPC, Redis, & HTTP
+libraries
+* Jaeger visualizations for distributed traces OpenTelemetry collector
+* Always on sampling (100% of telemetry is submitted) Synthetic load generation
+
+## Current Architecture
+
+!["Screenshot of the current architecture."](../../../../static\img\blog\current-demo-architecture.png/)
```
Comment (austinlparker) [inline content/en/blog/2022/demo-announcement.md:57]:
```suggestion
!["Screenshot of the current architecture."](/img/blog/current-demo-architecture.png/)
```

File: content/en/blog/2022/demo-announcement.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+/current-demo-architecture.png "Screenshot of the current architecture")
```
Comment (austinlparker) [inline content/en/blog/2022/demo-announcement.md:59]: Delete this and add the alt text to ln57

File: content/en/blog/2022/demo-announcement.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+etc. \
+* An enhanced OpenTelemetry collector gateway capabilities for ingestion,
+  transformation, and export
+* Probability based sampling
+* Feature flag service to demonstrate various scenarios like fault injection &
+  how to emit telemetry from a feature flag reliant service
+
+## Future Architecture
+
+!["Screenshot of the future architecture."](../../../../static\img\blog\future-demo-architecture.png "Screenshot of the future architecture")
```
Comment (austinlparker) [inline content/en/blog/2022/demo-announcement.md:87]:
```suggestion
!["Screenshot of the future architecture."](/img/blog/future-demo-architecture.png "Screenshot of the future architecture")
```

**Review by austinlparker — CHANGES_REQUESTED** [review]

File: content/en/blog/2022/demo-announcement.md (hunk 2 above)
Comment (cartersocha) [reply content/en/blog/2022/demo-announcement.md]: Okay I think everything is updated. Barring any other reviews it looks good from my end & ready to be scheduled to be published on Monday, the 20th. 

Semi unrelated but I'd be interested in any user stats after we release it

File: content/en/blog/2022/demo-announcement.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+### Project Goals
+
+- Provide developers with a robust sample application they can use in learning
+OpenTelemetry instrumentation.
+- Provide observability vendors with a single, well-supported, demo platform
+that they can further customize or simply use OOB.
+- Provide the OpenTelemetry community with a living artifact that demonstrates
+the features and capabilities of OTel APIs, SDKs, and tools.
+- Provide OpenTelemetry maintainers and working groups a platform to demonstrate
+new features/concepts real world like scenarios.
```
Comment (austinlparker) [inline content/en/blog/2022/demo-announcement.md:63]:
```suggestion
new features/concepts in real world like scenarios.
```

File: content/en/blog/2022/demo-announcement.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+the pre-existing feature set from the GCP demo remains the same:
+
+- 10 application microservice with support for 6 languages (C#, Go, Java,
+Node.js, Python, and Ruby)
+  - Ruby support was added within the last 2 weeks of publishing date
+- Designed to work on docker locally
+- Uses redis cache
+- Auto-instrumentation using instrumentation libraries Tracing support for the
+gRPC, Redis, and HTTP libraries
+- Jaeger visualizations for distributed traces OpenTelemetry collector
```
Comment (austinlparker) [inline content/en/blog/2022/demo-announcement.md:80]:
```suggestion
- Jaeger visualizations for distributed traces, forwarded by OpenTelemetry collector
```

File: content/en/blog/2022/demo-announcement.md [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+- Always on sampling (100% of telemetry is submitted) Synthetic load generation
```
Comment (austinlparker) [inline content/en/blog/2022/demo-announcement.md:81]:
```suggestion
- Always on sampling (100% of telemetry is submitted) and synthetic load generation
```

**Review by austinlparker — DISMISSED** [review]
nits/grammar

**Comment by cartersocha** [comment]
Thanks for the review! I can’t tell where the final suggested change blocking the repo checks is coming from. 

Is it just waiting for you to formally approve the pr?

**Review by austinlparker — APPROVED** [review]

**Comment by austinlparker** [comment]
I'll ago ahead and merge to get it live, we can talk it up starting Monday

**Pull request merged**